astroquery/esasky/core.py

```diff
@@ -0,0 +1,624 @@
+            for the chosen mission and object.
+            It is structured in a TableList like this:
+                TableList with 8 tables:
+                    '0:Gaia DR1 TGA' with 8 column(s) and 25 row(s) 
+                    '1:HSC' with 8 column(s) and 75 row(s) 
+        
+        Examples
+        --------
+        query_object_catalogs("m101", "all")
+        
+        import astropy.units as u
+        query_object_catalogs("265.05, 69.0", "Gaia DR1 TGA")
+        query_object_catalogs("265.05, 69.0", ["Gaia DR1 TGA", "HSC"])
+        """
+        self.query_region_catalogs(position, self.__ZERO_ARCMIN_STRING, catalogs, get_query_payload, cache)
+
+    def query_region_maps(self, position, radius, missions = __ALL_STRING, get_query_payload = False, cache = True):
+        """
+        This method queries a chosen region for all available maps
+        and returns a TableList with all the found maps metadata for the chosen missions and region.
+
+        Parameters
+        ----------
+        position : str or `astropy.coordinates` object
+            Can either be a string of the location, eg 'M51', or the coordinates of the object.
+        radius : str or `~astropy.units.Quantity`
+            The radius of a region.
+        missions : string or list, optional
+            Can be either a specific mission or a list of missions (all mission names are found in list_missions()) 
+            or 'all' to search in all missions. Defaults to 'all'. 
+        get_query_payload : bool, optional
+            When set to True the method returns the HTTP request parameters. Defaults to False.
+            
+        Returns
+        -------
+        table_list : `astroquery.utils.TableList`
+            Each mission returns a `astroquery.table.Table` with the metadata and observations available 
+            for the chosen missions and region.
+            It is structured in a TableList like this:
+                TableList with 8 tables:
+                    '0:HERSCHEL' with 8 column(s) and 25 row(s) 
+                    '1:HST' with 8 column(s) and 735 row(s) 
+        
+        Examples
+        --------
+        query_region_maps("m101", "14'", "all")
+        
+        import astropy.units as u
+        query_region_maps("265.05, 69.0", 14*u.arcmin, "Herschel")
+        query_region_maps("265.05, 69.0", ["Herschel", "HST"])
+        """        
+        sanatized_position = self._sanatize_input_position(position)
+        sanatized_radius = self._sanatize_input_radius(radius)
+        sanatized_missions = self._sanatize_input_mission(missions)
+                
+        query_result = {}
+
+        coordinates = commons.parse_coordinates(sanatized_position)
+            
+        self._store_query_result_maps(query_result, sanatized_missions, coordinates, sanatized_radius, get_query_payload, cache)
+        
+        if(get_query_payload):
```

Comment:
parentheses aren't necessary here, but they're acceptable.  Please add a space between `if` and `(`